### PR TITLE
Edited for Ethereum Classic

### PR DESCRIPTION
**Pull request by TheCrowbill**
All EIP references changed to ECIP.
Most links updated to ETC appropriate URL.
Minor grammatical corrections.
Minor punctuation corrections.
Minor content edits for consistency.

### CODE REVIEW

File: ECIP-1.sample.md [hunk 1]
```diff
@@ -1,102 +1,103 @@
...
 
-If the EIP collaborators approves, the EIP editor will assign the EIP a number, label it as Standards Track, Informational, or Process, give it status "Draft", and add it to the git repository. The EIP editor will not unreasonably deny an EIP. Reasons for denying EIP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the Ethereum philosophy.
+Once the champion has asked the Ethereum community as to whether an idea has any chance of acceptance, a draft ECIP should be presented as a Pull Request. This gives the author a chance to flesh out the draft ECIP; to make properly formatted, of high quality, and to address initial concerns about the proposal.
```
Comment (whilei) [inline ECIP-1.sample.md:37]: s/Ethereum/Ethereum Classic

**Review by whilei — APPROVED** [review]

**Comment by whilei** [comment]
Lookin' good @TheCrowbill. Sorry it took a while for a review. Thanks!

**Comment by TheCrowbill** [comment]
If this is approved, would someone who has write access please merge this PR? Thanks!

**Comment by pyskell** [comment]
Trusting @whilei's judgement here and merging. Also read over most of it myself and it looks good to me.

Feel free to revert if necessary.

**Pull request merged**